faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -163,6 +165,16 @@ object DateV {
   def apply(value: String): DateV = DateV(LocalDate.parse(value))
 }
 
+case class BytesV(@(JsonIgnore @param @field @getter) bytes: IndexedSeq[Byte]) extends ScalarValue {
+  @JsonProperty("@bytes")
+  lazy val strValue = BaseEncoding.base64Url().encode(bytes.toArray)
+}
+object BytesV {
+  def apply(bytes: Array[Byte]): BytesV = BytesV(bytes.toIndexedSeq)
+  def apply(bytes: Int*): BytesV = BytesV(bytes map { _.toByte } toIndexedSeq)
```

Comment:
Shouldn't be a sequence of bytes? I'm trying to understand when this conversion is useful.